canalgen/config.json

```
@@ -0,0 +1,13 @@
+{
+  "NODE": "BMS",
+  "IGNORED_FILES": [
```

Let's rename this to make it more clear
```suggestion
  "IGNORED_DBCS": [
```